helm/idol-nifi/templates/nifi/nifi.yaml

```diff
@@ -55,14 +55,19 @@ spec:
         - |
           if [ -z "$(ls -A -- /mnt/store/nifi/conf)" ]; then
             echo Copying files into conf directory
-            cp -r /opt/nifi/nifi-current/conf/ /mnt/store/nifi/
+            cp -rv /opt/nifi/nifi-current/conf/ /mnt/store/nifi/
           fi
-
+          echo Copying files into extensions directory
+          cp -rnv /opt/nifi/nifi-current/extensions/ /mnt/store/nifi/
```

Comment:
Do we need this in `ensure-conf` now?

Reply:
From our off ticket conversation:
> definitely still want persisting extensions directory 
> that at least lets you write a prestart script that checks if your required extensions are present before downloading?
> downloading/copying

I think that it is still useful to persist this directory, for the reasons you stated above, because it also maintains any NARs manually copied into that directory, and because it would be difficult for the end user to achieve the same behavior because providing their own mount for `/opt/nifi/nifi-current/extensions` would overwrite the originally provided NARs.